conf/documentation.conf

```diff
@@ -632,8 +632,8 @@ EOT
 [alerting.emailaddr]
 type=text
 description=<<EOT
-Email address to which notifications of rogue DHCP servers, violations 
-with an action of "email", or any other PacketFence-related message 
+Comma-delimited list of email addresses to which notifications of rogue DHCP servers, violations 
```

Comment:
Shouldn't we write "to which notifications of rogue DHCP servers ... should be sent, or or any other PacketFence-related message." ?

Reply:
I kept the end of text in a separated line